Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+<table>
+  <tr>
+    <td style="min-width: 25em;">__TOC__</td>
+    <td style="vertical-align: top; padding-left: 1em;">
+'''STATUS: READY'''
+'''RRA Version supported: 2.5.0'''
+
+Any team uses a risk based methodology when making any decision.
+The Rapid Risk Asessment or Rapid Risk Analysis (RRA) process helps formalize these decisions and ensures that they're
+reproducible, consistent and easy to communicate.
+
+This document assumes that the reader is familiar with risk definition, and the Mozilla risk levels from the
+[Risk_management] documentation (i.e. read it first).
+
+The Enterprise Information Security Team (Infosec) maintains this document as a reference guide for security engineers
+and anyone interested in following our risk management processes.
+
+Updates to this page should be submitted to the [https://github.com/mozilla/wikimo_opsec/ source repository on github].
+Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/master commit history].
+
+<span style="float: right; padding-top: 3em;">[[File:OpSec.png|300px]]</span>
+    </td>
+  </tr>
+</table>
+
+= Rapid Risk Assessment =
+
+{| class="wikitable"
+|-
+! <span style="color:red;">'''WARNING'''</span>
+|-
+|
+The Rapid Risk Analysis/Assessment (RRA) is not a security review or a full threat-model. A threat model, review, pen-testing, etc. may
+however follow an RRA.
+The RRA strongly focuses on getting the maximum amount of actionable information in the shortest amount of time available.
```

Comment:
The RRA focuses on capturing as much information as possible about the risk posture of a service or product in a short amount of time, typically 30 minutes.